tuf/ngclient/updater.py

```diff
@@ -57,23 +57,23 @@
 
 
 class Updater:
-    """Creates a new Updater instance and loads trusted root metadata.
+    """Creates a new ``Updater`` instance and loads trusted root metadata.
 
     Args:
         metadata_dir: Local metadata directory. Directory must be
-            writable and it must contain a trusted root.json file.
+            writable and it must contain a trusted ``root.json`` file
```

Comment:
Not sure if we need to backtick-wrap file names? Maybe just quote...?
```suggestion
            writable and it must contain a trusted 'root.json' file
```